proto/kaskada/kaskada/v1alpha/common.proto

```diff
@@ -87,6 +88,9 @@ message TableConfig {
   // TODO: Consider instead defaulting to a name derived from the type of the group
   // key. This would default to allowing implicit grouping amongst tables.
   string grouping = 5;
+
+  // The backing source for the table
+  Source source = 6;
```

Comment:
this is identical to the `Source` property used `wren`'s API for creating a new table.  For stream-backed tables it would include all the connection and authentication info we need to connect to the stream.